### PR TITLE
Update astropy to 4.0.2 to fix fits 1d issue

### PR DESCRIPTION
**Pull request by maxnoe**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -29,7 +29,7 @@
     packages=find_packages(),
     include_package_data=True,
     install_requires=[
-        "astropy~=4.0",
+        "astropy~=4.0,>=4.0.2",
```
Comment (HealthyPear) [inline setup.py:32]: why keeping `~=4.0` if 4.0.2 is the minimal version that solves this probem?

**Review by HealthyPear — CHANGES_REQUESTED** [review]
We should also remove any related material from the code, e.g. in the integration test we have code checking the rows dimentionality in the HDUs that will become stale with these changes.

File: setup.py (hunk 1 above)
Comment (maxnoe) [reply setup.py:32]: Because `~=` means any 4.x version. So this means "any 4.x" version larger than `4.0.2`

**Comment by maxnoe** [comment]
Ok, I removed the "ignored_errors" part in the integration test as these are now expected to pass and they do

**Review by HealthyPear — APPROVED** [review]

**Pull request merged**